cli/schema/cypress.schema.json

```
@@ -244,8 +244,7 @@
             "system",
             "bundled"
           ],
-          "default": "bundled",
-          "description": "If set to 'system', Cypress will try to find a Node.js executable on your path to use when executing your plugins. Otherwise, Cypress will use the Node version bundled with Cypress."
+          "description": "If set to 'system', Cypress will use the node version that was used to launch the cli when executing your plugins. Otherwise, Cypress will use the Node version bundled with Cypress."
```

Shouldn't this have 'system' as the default?

This description should be reversed also since the logic would apply if set to 'bundled'. 

```suggestion
          "description": "If set to 'bundled', Cypress will use the Node version bundled with Cypress. Otherwise, Cypress will use the Node version that was used to launch the Cypress. This Node version is used when executing your plugins file and building spec files."
```

---

Technically there is no default now because adding either `system` or `bundled` will trigger the deprecation warning. 

https://github.com/cypress-io/cypress/blob/9.0-release/packages/server/lib/config_options.ts#L153-L155

I'm not sure exactly how to use this file, but I didn't want to encourage people to add the 'nodeVersion' option to config.

---

Actually should I remove the `nodeVersion` entry entirely because it's deprecated?

---

But the deprecation warning is just a warning - the value still works. I don't understand the blocker for showing the default value as it is in the schema.

---

If things are deprecated they are still documented as before because the value is still completely functional, it just warns that it will be removed later.

---

We met in standup and decided to restore the default and mark the option as deprecated in the description.